README.md

```diff
@@ -1,14 +1,14 @@
 # Staff Chat
-[<img src="https://img.shields.io/badge/Poggit-view-brightgreen.svg" width="100" height="25" />](https://poggit.pmmp.io/ci/ThunderDoesPlugins/StaffChat/StaffChat)
-[<img src="https://img.shields.io/badge/Discord-join-697EC4.svg" width="100" height="25" />](https://discord.gg/uBghvNp)
+[<img src="https://img.shields.io/badge/Poggit-view-brightgreen.svg" width="95" height="20" />](https://poggit.pmmp.io/ci/ThunderDoesPlugins/StaffChat/StaffChat)
```

Comment:
The IMG tags are there because i prefer them to be big then small, also please dont stretch the images, mine is not random numbers that i think look good but it's calculated to be proportional